android/jni/app-android.cpp

```diff
@@ -650,7 +655,7 @@ extern "C" void Java_org_ppsspp_ppsspp_NativeApp_init
 	EARLY_LOG("NativeApp.init() -- begin");
 	PROFILE_INIT();
 
-	std::lock_guard<std::mutex> guard(renderLock);
+	std::lock_guard<std::mutex> guard(renderLock);  // Note: This is held for the rest of this function - intended?
```

Comment:
This only overlaps with shutdown and render, neither of which should start happening until init is done, IMHO.  So it seems like a good thing for this lock to last the entire time.

That said, maybe we could run some more on the background or in a thread.  I don't really think that would change this lock lifetime itself, but rather move other things out of this func directly.

-[Unknown]